components/Header/HeaderMovile.tsx

```diff
@@ -0,0 +1,20 @@
+import MenuMovile from "./MenuMovile";
+
+
+const HeaderMovile = () => {
+    return (
+        <div className=' lg:hidden'>
+            <div className='relative flex justify-between bg-primary text-white font-bold text-center h-12'>
```

Comment:
Add the necessary classes to properly center the text to match the logo icon on the right.

![image](https://github.com/disnelrr/mivetonline/assets/35430665/2d8520a7-8c5f-4ebe-99a1-6e84a2830bf7)